doc/usage.rst

```diff
@@ -872,3 +872,20 @@ like this:
   $ labgrid-client -p example allow sirius/john
 
 To remove the allow it is currently necessary to unlock and lock the place.
+
+Using Places Within a Session
```

Comment:
Most of the other titles are also upper case, like "Remote Access"

Reply:
You're right, it's not consistent.

I went to look into the [rendered markdown](https://github.com/ARM-software/labgrid/blob/b17e17a0a8a18bab291b9857131fc55389a161dd/doc/usage.rst), and it seems that we're using the wrong heading level. I guess we want it to be a level 3, so we need `^` instead of `-`.